src/patternfly/components/Tabs/tabs.scss

```
@@ -23,6 +23,11 @@ $pf-c-tabs--spacer-map: build-spacer-map("none", "sm", "md", "lg", "xl", "2xl");
   --pf-c-tabs--m-color-scheme--light-300__link--BackgroundColor: transparent;
   --pf-c-tabs--m-color-scheme--light-300__item--m-current__link--BackgroundColor: var(--pf-global--BackgroundColor--light-300);
 
+  // Tab list
+  --pf-c-tabs__list--ScrollSnapTypeAxis: x;
+  --pf-c-tabs__list--ScrollSnapTypeBehavior: proximity;
```

Sorry I suggested the wrong thing. According to the [snap spec](https://drafts.csswg.org/css-scroll-snap-1/#snap-strictness) we should call this "strictness"

```suggestion
  --pf-c-tabs__list--ScrollSnapTypeStrictness: proximity;
```